recipes/py_dempster_shafer/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "py_dempster_shafer" %}
+{% set version = "0.7" %}
+{% set sha256 = "273e62da43a589ddcb6a3b6f9aab0bbebafbd623ed8f75e442360711de6ba633" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --record record.txt
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - pyds
+
+about:
+  home: https://github.com/reineking/pyds
+  license: BSD-3-Clause
+  license_family: BSD
+  # license_file is missing in PyPI package. Requested it here: https://github.com/reineking/pyds/issues/3
```

Comment:
Can you please manually package the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
Sure, just did this!